e2e/tests/services.rs

```diff
@@ -161,12 +162,16 @@ impl OrderbookServices {
             Duration::from_secs(120),
         ));
 
+        let registry = Registry::default();
+        let metrics = Arc::new(Metrics::new(&registry).unwrap());
```

Comment:
We have NoopMetrics for the e2e tests.

Reply:
True but this also serves as another test that there is no panic in the real metrics code path. Not sure which is better.